public/components/integrations/home.tsx

```diff
@@ -97,15 +97,32 @@ export const Home = (props: HomeProps) => {
           <Route
             exact
             path={'/available/:id/setup'}
-            render={(routerProps) => (
-              <SetupIntegrationPage
-                integration={decodeURIComponent(routerProps.match.params.id)}
-                dataSourceManagement={dataSourceManagement}
-                notifications={notifications}
-                dataSourceEnabled={dataSourceEnabled}
-                savedObjectsMDSClient={savedObjectsMDSClient}
-              />
-            )}
+            render={(routerProps) => {
+              const integrationId = decodeURIComponent(routerProps.match.params.id);
+              const capitalizedIntegrationId =
+                integrationId.charAt(0).toUpperCase() + integrationId.slice(1);
+
+              props.chrome.setBreadcrumbs([
+                {
+                  text: 'Integrations',
+                  href: '#/available',
+                },
+                {
+                  text: capitalizedIntegrationId,
+                  href: `#/available/${integrationId}`,
+                },
+              ]);
+
+              return (
+                <SetupIntegrationPage
+                  integration={integrationId}
+                  dataSourceManagement={props.dataSourceManagement}
```

Comment:
This may not relate to your work but seems like dataSourceManagement and dataSourceEnabled are passing down through the entire component chain without using it until lower level component, we should consider to expose them as context or some other similar way.